### PR TITLE
terraform/cloudflare: refactor

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

**Comment by zimbatm** [comment]
Yes, it looks functionally correct. I'm not sure what prompted the move. Was the code getting in the way of something else?

**Comment by zowoq** [comment]
Now that we're going to be advertising the hydra jobsets I thought it could do with a refactor, it is verbose but not very readable.

**Comment by Mic92** [comment]
From my personal experience a little bit more verbosity in terraform is better than too much abstraction. Infrastructure automation is just different from programming because it also involves state management. i.e. for targeted applies its nice if one can copy'n'paste target names. I am sure this is still possible, but it just requires more effort and reading now. But feel free to follow whichever branch you want.

File: terraform/cloudflare.tf [hunk 1]
```diff
@@ -4,7 +4,10 @@ locals {
     for role in data.cloudflare_account_roles.account_roles.roles :
     role.name => role
   }
-  cf_admins = local.admins
+  cf_admins = {
+    for key, value in local.admins :
+    key => value if key != "adisbladis"
```
Comment (zowoq) [inline terraform/cloudflare.tf:9]: Skipping this invite as we keep resending it.

**Pull request merged**